DigitalLearningSolutions.Web/ViewModels/LearningMenu/TutorialViewModel.cs

```diff
@@ -7,39 +7,65 @@
 
     public class TutorialViewModel
     {
-        public TutorialInformation TutorialInformation { get; }
+        public string TutorialName { get; }
+        public string CourseTitle { get; }
+        public string Status { get; }
+        public string? TutorialPath { get; }
+        public string? VideoPath { get; }
         public string? Objectives { get; }
         public int CustomisationId { get; }
         public int SectionId { get; }
+        public int TutorialId { get; }
         public string? SupportingMaterialPath { get; }
 
+        private readonly int timeSpent;
+        private readonly int averageTutorialDuration;
+        private readonly int currentScore;
+        private readonly int possibleScore;
+        private readonly bool canShowDiagnosticStatus;
+        private readonly int attemptCount;
+
         public TutorialViewModel(
             IConfiguration config,
             TutorialInformation tutorialInformation,
             int customisationId,
-            int sectionId
+            int sectionId,
+            int tutorialId
         )
         {
-            TutorialInformation = tutorialInformation;
+            TutorialName = tutorialInformation.Name;
+            CourseTitle = tutorialInformation.CourseTitle;
+            TutorialPath = tutorialInformation.TutorialPath;
+            VideoPath = tutorialInformation.VideoPath;
+            Status = tutorialInformation.Status;
+
+            timeSpent = tutorialInformation.TimeSpent;
+            averageTutorialDuration = tutorialInformation.AverageTutorialDuration;
+            currentScore = tutorialInformation.CurrentScore;
+            possibleScore = tutorialInformation.PossibleScore;
+            canShowDiagnosticStatus = tutorialInformation.CanShowDiagnosticStatus;
+            attemptCount = tutorialInformation.AttemptCount;
+
             CustomisationId = customisationId;
             SectionId = sectionId;
+            TutorialId = tutorialId;

```

Comment:
Isn't the tutorial id part of the tutorial information object? 🙂

Reply:
It is, sorry! Fixed in 719b12c 🙂